src/cdev/commands/initializer.py

```diff
@@ -58,14 +61,26 @@ def create_project_cli(args) -> None:
         template_name = None
```

Comment:
You can make this code more readable by simply assigning the template_name = args.template
And checking if it is not None and it is in the list of valid templates